### PR TITLE
Add Integration Instance debug logging docs.

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/34447

## Description
Adds documentation for the instance debug logging feature.

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@amshamah419 Doc review completed.

**Comment by glicht** [comment]
@amshamah419 we should clarify that this mode is also very useful for long running integrations such as EDL or TAXII-Server. Basically it helps troubleshooting when it is not possible to run the desired command in `debug-mode` from the playground. Either it is a long running integration or the issue occurs from time to time such as with `Fetch Incidents`.

**Comment by ShirleyDenkberg** [comment]
@amshamah419 @glicht added Guy's comment.

File: content-repo/extra-docs/articles/troubleshooting.md [hunk 1]
```diff
@@ -181,6 +181,36 @@ For example for an integration instance name of: `Cortex_XDR_instance_1` run the
...
 
 Screenshot of running a `test-module` command with `debug-mode=true` and the resulting log file (`test-module.log`):
-![test-module-debug](../../../docs/doc_imgs/reference/test-module-debug.png)   
 
+![test-module-debug](../../../docs/doc_imgs/reference/test-module-debug.png)
+
+## Integration Debug Logs
+Starting with version 6.2, it is possible to create logs for an instance of an integration in order to get debug information for a specific instance over a period of time. 
+
+This mode is especially useful for long running integrations such as EDL or TAXII-Server. Basically it helps troubleshooting when it is not possible to run the desired command in debug-mode from the playground. Either it is a long running integration or the issue occurs from time to time such as with the ***fetch-incidents*** command.
```
Comment (glicht) [inline content-repo/extra-docs/articles/troubleshooting.md:190]:
```suggestion
This mode is especially useful for long running integrations such as EDL or TAXII-Server. Basically it helps troubleshooting when it is not possible to run the desired command in `debug-mode` from the playground. Either it is a long running integration or the issue occurs from time to time such as with the ***fetch-incidents*** command.
```

**Review by glicht — APPROVED** [review]

File: content-repo/extra-docs/articles/troubleshooting.md (hunk 1 above)
Comment (amshamah419) [reply content-repo/extra-docs/articles/troubleshooting.md:190]:
```suggestion
This mode is especially useful for long running integrations such as EDL or TAXII-Server. It helps troubleshooting when it is not possible to run the desired command in `debug-mode` from the playground. Whether it is a long running integration, or the issue occurs from time to time such as with the ***fetch-incidents*** command.
```

**Pull request merged**